static/app/actionCreators/dashboards.tsx

```diff
@@ -239,6 +217,43 @@ export function validateWidgetRequest(
   ] as const;
 }
 
+export function updateDashboardPermissions(
+  api: Client,
+  orgId: string,
+  dashboard: DashboardDetails
+): Promise<DashboardDetails> {
+  const {projects, environment, permissions} = dashboard;
+  const data = {
+    projects,
```

Comment:
these fields seem to be necessary to pass some backend validation

Reply:
Let's just add a small note to explain why that's there. Just curious, what's the error you're seeing? Seems like it's something that we shouldn't be throwing